scripts/community/filedetails_guide.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+if( document.querySelector( '.guideTopContent' ) )
+{
+	const guide = document.querySelector( '.guide' );
+	const spoilers = guide.querySelectorAll( '.bb_spoiler' );
```

Comment:
I'd move this inside of the change event, so that a list of references is not kept. You can change the if to `guide.querySelector( '.bb_spoiler' );` to still check if it has any.